iam_server.tf

```diff
@@ -40,7 +40,29 @@ locals {
         Action   = ["s3:AbortMultipartUpload", "s3:PutObject"]
         Resource = [local.large_queue_messages_bucket_arn, "${local.large_queue_messages_bucket_arn}/*"]
       }
-    ]
+      ],
+      local.has_sqs_queues ? [{
+        Effect = "Allow",
+        Action = [
+          "sqs:SendMessage"
+        ],
+        Resource = [
+          var.sqs_queues.webhooks
+        ]
+      }] : [],
+      local.has_iot_topic ? [{
+        Effect = "Allow",
+        Action = [
+          "iot:AttachPolicy",
+          "iot:AttachPrincipalPolicy",
+          "iot:CreateCertificateFromCsr",
+          "iot:RegisterCertificateWithoutCA",
+          "iot:CreateKeysAndCertificate",
+          "iot:CreatePolicy"
+        ],
+        Resource = ["*"],
+      }] : []
+    )
   })
 
   drain_and_server_policy = jsonencode({
```

Comment:
Maybe we should move `drain_and_server_policy` into a separate file. It confused me for a second because it was in the iam_server.tf file.

Reply:
agreed. let me move it.